### PR TITLE
Config for running data ablations

### PR DESCRIPTION
**Pull request by dirkgr**

### CODE REVIEW

File: configs/mcli/olmo7-ablation.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: olmo7-ablation  # can't have "_" or "." here
+image: mosaicml/pytorch:2.1.2_cu121-python3.10-ubuntu20.04
+compute:
+  gpus: 64
+  cluster: r12z3
+  gpu_type: a100_40gb
+integrations:
+  - integration_type: git_repo
+    git_repo: allenai/LLM
```
Comment (2015aroras) [inline configs/mcli/olmo7-ablation.yaml:9]: nit: Change this and the later `cd` from `LLM` to `OLMo`

**Review by 2015aroras — APPROVED** [review]

File: configs/mcli/olmo7-ablation.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: olmo7-ablation  # can't have "_" or "." here
+image: mosaicml/pytorch:2.1.2_cu121-python3.10-ubuntu20.04
+compute:
+  gpus: 64
+  cluster: r12z3
```
Comment (epwalsh) [inline configs/mcli/olmo7-ablation.yaml:5]: This should be `r7z2` now (see message from MosaicML folks on Slack).
```suggestion
  cluster: r7z2
```

File: configs/olmo7-ablation.yaml [hunk 3]
```diff
@@ -0,0 +1,641 @@
...
+    data:
+      num_workers: 0
+      drop_last: true
+      # pin_memory: true
+      # prefetch_factor: 1
+      # persistent_workers: false
+      # timeout: 0
+      datasets:
+        4chan-validation:
+        - s3://ai2-llm/eval-data/perplexity/v2_small_gptneox20b/4chan/val.npy
```
Comment (epwalsh) [inline configs/olmo7-ablation.yaml:110]: There's a newer version of all of these. See `configs/official/OLMo-7B.yaml` for example.

File: scripts/train.py [hunk 4]
```diff
@@ -240,6 +241,7 @@ def dummy_init_fn(module: torch.nn.Module) -> None:
 
 
 if __name__ == "__main__":
+    mp.set_start_method("spawn")
```
[the remark this answers is not in the document]
Comment (dirkgr) [reply scripts/train.py:244]: Yes, it is.

File: olmo/util.py [hunk 5]
```diff
@@ -549,7 +549,7 @@ def _s3_get_bytes_range(
             )
         except boto_exceptions.ClientError as e:
             if int(e.response["Error"]["Code"]) == 404:
-                raise FileNotFoundError(f"s3://{bucket_name}/{key}") from e
+                raise FileNotFoundError(f"{scheme}://{bucket_name}/{key}") from e
```
[the remark this answers is not in the document]
Comment (dirkgr) [reply olmo/util.py:552]: Oh, I did actually get them all, just not in the commit that you were looking at.

File: configs/mcli/olmo7-ablation.yaml (hunk 2 above)
Comment (dirkgr) [reply configs/mcli/olmo7-ablation.yaml]: I've been switching back and forth depending on what's available.

File: configs/olmo7-ablation.yaml (hunk 3 above)
Comment (dirkgr) [reply configs/olmo7-ablation.yaml]: Now that I've run the baseline with these, I don't want to change it anymore.

**Review by epwalsh — APPROVED** [review]

**Pull request merged**